proposals/p2875.md

```diff
@@ -138,9 +170,9 @@ permitted.
 
 ### Direct calls
 
-When the callee in a call expression can be resolved to a declaration, either
-because it is the name of a parameterized entity like a generic class or
-interface, or because it has a function type, the call is a _direct call_.
+When the callee in a call expression is either the name of a parameterized
+entity like a generic class or interface or has a function type or bound method
+type, the call is a _direct call_.
```

Comment:
I found this a bit hard to parse since it has the word "or" three times, with different precedences. I think you mean:

> When the callee in a call expression is either (the name of a parameterized entity like a generic (class or interface)) or (has a (function type or bound method type)), the call is a _direct call_.

I think this may just be a bit too complicated of a sentence structure. Maybe use bullets to make this clearer?

```suggestion
A call expression is a _direct call_ when the callee is:

-    the name of a parameterized entity, like a generic class or interface,
-    has a function type, or
-    has bound method type.
```